crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -208,7 +208,7 @@ pub(crate) trait StateReadExt: StateRead + crate::assets::StateReadExt {
         Ok(balance)
     }
 
-    #[instrument(skip_all)]
+    #[instrument(skip_all, err)]
```

Comment:
Note on this `T: AddressBytes` and all methods that contain a similar argument: in https://github.com/astriaorg/astria/pull/1495 I have added logic to display the address. It would be very useful to get that context here.

Reply:
Tried to add similar fields wherever valuable in 2b30bf2 :)